chainer/functions/connection/deconvolution_nd.py

```diff
@@ -303,12 +303,12 @@ def deconvolution_nd(x, W, b=None, stride=1, pad=0, outsize=None,
     weight ``W``, and the bias vector ``b``.
 
     Args:
-        x (chainer.Variable or :class:`numpy.ndarray` or cupy.ndarray):
-            Input variable of shape :math:`(n, c_I, d_1, d_2, ..., d_N)`.
-        W (chainer.Variable or :class:`numpy.ndarray` or cupy.ndarray):
-            Weight variable of shape :math:`(c_O, c_I, k_1, k_2, ..., k_N)`.
-        b (chainer.Variable or :class:`numpy.ndarray` or cupy.ndarray):
-            Bias variable of length :math:`c_O` (optional).
+        x (~chainer.Variable or :class:`numpy.ndarray` or cupy.ndarray):
```

Comment:
As you can see by compiling the document, `~`  does not work in this case (Napoleon cannot seem to recognize it as a class when the classname is followed by some words).